migrations/20180629130216_add_order_to_sections.js

```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex) {
+  return knex.schema.table("Sections", table => {
+    table
+      .integer("order")
+      .defaultsTo(0)
+      .notNullable();
+  });

```

Comment:
Think we need to retroactively add section order to previously created sections here as otherwise all sections that already exist will suddenly have an order of 0.

Reply:
New migration added 'seed_section_order_columns'